modules/data-table/test/spec.coffee

```diff
@@ -266,19 +275,16 @@ describe 'data-table', ->
           hx.select(rows[2]).selectAll('td').selectAll('.hx-data-table-cell-key').text().should.eql(headers)
           hx.select(rows[2]).selectAll('td').selectAll('.hx-data-table-cell-value').text().should.eql(['Dan', '41', 'Builder'])
 
-        it 'should show the paginator block', ->
-          container.select('.hx-data-table-pagination-block-hidden').size().should.equal(0)
-
         it 'should not show the paginator', ->
-          container.select('.hx-data-table-multi-page').size().should.equal(0)
+          container.selectAll('.hx-data-table-paginator-visible').size().should.equal(0)
```

Comment:
I've standardised some classes - it was difficult to follow what was going on in some places and in others the structure has changed so the old classes didn't make sense.
 
In this case, there are now 3 paginators that are made visible by adding this class to them individually to make it easier to manage